config/src/main/java/org/springframework/security/config/annotation/web/configurers/RememberMeConfigurer.java

```diff
@@ -267,6 +267,11 @@ public void init(H http) throws Exception {
 		validateInput();
 		String key = getKey();
 		RememberMeServices rememberMeServices = getRememberMeServices(http, key);
+		if (key == null) {
+			if (rememberMeServices instanceof AbstractRememberMeServices) {
+			    key = ((AbstractRememberMeServices) rememberMeServices).getKey();
```

Comment:
Please use a tab for indentation instead of spaces.

Reply:
Done